pkg/lib/registry/registry_test.go

```diff
@@ -402,49 +400,42 @@ func TestCheckForBundles(t *testing.T) {
 						},
 					},
 					action: actionAdd,
-					expected: map[string]*registry.Package{
-						"testpkg": {
-							Name: "testpkg",
-							Channels: map[string]registry.Channel{
-								"alpha": {
-									Nodes: map[registry.BundleKey]map[registry.BundleKey]struct{}{
-										registry.BundleKey{
-											BundlePath: "unorderedReplaces-1.0.0",
-											Version:    "1.0.0",
-											CsvName:    "unorderedReplaces-1.0.0",
-										}: nil,
-										registry.BundleKey{
-											BundlePath: "unorderedReplaces-1.1.0",
-											Version:    "1.1.0",
-											CsvName:    "unorderedReplaces-1.1.0",
-										}: nil,
-										registry.BundleKey{
-											BundlePath: "unorderedReplaces-1.2.0",
-											Version:    "1.2.0",
-											CsvName:    "unorderedReplaces-1.2.0",
-										}: nil,
-									},
-								},
-								"stable": {
-									Nodes: map[registry.BundleKey]map[registry.BundleKey]struct{}{
-										registry.BundleKey{
-											BundlePath: "unorderedReplaces-1.0.0",
-											Version:    "1.0.0",
-											CsvName:    "unorderedReplaces-1.0.0",
-										}: nil,
-										registry.BundleKey{
-											BundlePath: "unorderedReplaces-1.1.0",
-											Version:    "1.1.0",
-											CsvName:    "unorderedReplaces-1.1.0",
-										}: nil,
-									},
-								},
+				},
+			},
+			wantErr: fmt.Errorf("add prunes bundle unorderedReplaces-1.0.0 (unorderedReplaces-1.0.0) from package testpkg, channel stable: this may be due to incorrect channel head (unorderedReplaces-1.1.0, skips/replaces [])"),
+		},
+		{
+			description: "silentPruneForExistingBundle",
+			steps: []step{
```

Comment:
Should we also test for the case where we're inserting multiple new bundles into the same graph? Both the failure case (one of those bundles doesn't replace another) and the success case (one of the bundles resets the graph, another gets added on top of it)?